app_feup/lib/view/widgets/ExamCard.dart

```diff
@@ -0,0 +1,179 @@
+            width: double.infinity,
+            decoration: BoxDecoration(
+              border: Border(
+                bottom: BorderSide(
+                  color: Theme.of(context).accentColor,
+                  width: 0.5
+                ),
+                top: BorderSide(
+                  color: Theme.of(context).accentColor,
+                  width: 0.5
+                )
+              )
+            ),
+          ),
+          new Center(
+            child: new Container(
+              padding: EdgeInsets.only(left: 12.0),
+              margin: EdgeInsets.only(top: 5.0),
+              child: new Row(
+                mainAxisSize: MainAxisSize.max,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  new Column(
+                    children: <Widget>[
+                      new Container(
+                        child: Text(this.secondExam.begin,style: Theme.of(context).textTheme.body2),
+                        margin: EdgeInsets.only(top: 5.0),
+                      ),new Container(
+                        child: Text(this.secondExam.end,style: Theme.of(context).textTheme.body2),
+                        margin: EdgeInsets.only(top: 24.0),
+                      ),
+                    ],
+                  ),
+                  new ConstrainedBox(
+                    constraints: new BoxConstraints(
+                        minWidth: 210,
+                        minHeight: 30,
+                        maxWidth: 300,
+                        maxHeight: 65
+                    ),
+                    child: new Container(
+                      margin: EdgeInsets.only(left: 24.0, top: 2.0),
+                      decoration: BoxDecoration(
+                        color: Theme.of(context).primaryColor,
+                        borderRadius: BorderRadius.all(Radius.circular(this.borderRadius)),
+                      ),
+                      child: Column(
+                        mainAxisSize: MainAxisSize.max,
+                        children: <Widget>[
+                          new Container(
+                            margin: EdgeInsets.only(top: 5.0),
+                            child: new Text(this.secondExam.subject, style: TextStyle(color: Theme.of(context).accentColor),),
+                          ),
+                          new Container(
+                            margin: EdgeInsets.only(top: 13.0),
+                            child: new Text(this.secondExam.rooms, style: TextStyle(color: Theme.of(context).accentColor),),
+                          )
+                        ],
+                      ),
+                    ),
+                  ),
+
+                ],
+              ),
+             )
+          )
+
+        ],
+      )
+    );
+  }
+}
+Future<void> getExams() async{
+  List<Exam> meias = await examsGet("https://sigarra.up.pt/feup/pt/exa_geral.mapa_de_exames?p_curso_id=742");
+  meias[0].printExam();
+
+}
```

Comment:
The getExams() method should be declared in the "controller" package

Reply:
This was just a test function, now removed.